examples/CSharp/Tutorial 1.1 - Hello Window/Tutorial 1.1 - Hello Window.csproj

```diff
@@ -10,4 +10,8 @@
       <PackageReference Include="Silk.NET" Version="1.1.1" />
```

Comment:
Could be wrong but GitHub mobile seems to think that you have a package reference and a project reference to silk in this project file, whereas we only need the latter.

Reply:
Strange.  The solution explorer showed no nuget packages but the XML was still in the csproj file.

VS bug?

Issue fixed